src/components/capture/Edit.tsx

```diff
@@ -160,62 +162,72 @@ function CaptureEdit() {
                 header: authenticatedRoutes.captures.edit.title,
             }}
         >
-            <EndpointConfigHydrator>
-                <ResourceConfigHydrator>
-                    <EntityEdit
-                        title="browserTitle.captureEdit"
-                        entityType={entityType}
-                        readOnly={{ detailsForm: true }}
-                        draftSpecMetadata={draftSpecsMetadata}
-                        callFailed={helpers.callFailed}
-                        resetState={resetState}
-                        errorSummary={
-                            <ValidationErrorSummary
-                                errorsExist={detailsFormErrorsExist}
-                            />
-                        }
-                        toolbar={
-                            <EntityToolbar
-                                GenerateButton={
-                                    <CaptureGenerateButton
-                                        entityType={entityType}
-                                        disabled={!hasConnectors}
-                                        callFailed={helpers.callFailed}
-                                        postGenerateMutate={mutateDraftSpecs}
+            <DraftInitializer>
```

Comment:
Addition of an edit draft initializer and details form store hydrator. No additional changes were made to the remainder of this file.